mfthomps/CyberCIEGE-port-to-Unity3d
# issue/169-impl-scrap-computer

Closes #169

Note: Requires new [.exe](https://nps.box.com/s/xn88owrsvj0fhskmf10coqz83vccn6dd)

I think the scrapping of computers now completely works.

To verify:
- Buy a computer and place on a WorkSpace
- From the Component dialog, select that computer and hit the Scrap button.
- Notice:
  - The Computer list changes
  - The GameObject is removed
  - There is no currently selected object
- Purchase a replacement computer for the same workspace.

Note: buying a computer doesn't cost money (#179), but selling it gives you a refund.
